pawn/Resources/Callbacks/OnPlayer/OnPlayerDeath.pwn

```diff
@@ -44,7 +44,7 @@ OnPlayerLameKill(playerId, killerId) {
             Player(killerId)->nicknameString(), lameKills);
     else
         format(message, sizeof(message),
-            "* %s has {A9C4E4}committed a driveby{CCCCCC} on %s! %s has {A9C4E4}%d lame kills{CCCCCC}.",
+            "* %s has {A9C4E4}committed a driveby{CCCCCC} on %s!",
             Player(killerId)->nicknameString(), Player(playerId)->nicknameString(),
             Player(killerId)->nicknameString(), lameKills);
```

Comment:
Hey @XilentLVP, Awesome work man! Remove this line too!
     
```suggestion
            Player(killerId)->nicknameString(), Player(playerId)->nicknameString());
```
 This one since you removed the %s & %d you have to remove these too, if I am right.